src/components/PageList/Item.vue

```diff
@@ -449,4 +463,10 @@ export default {
 	right: 0;
 	margin: 0;
 }
+
+@keyframes highlight-animation {
```

Comment:
Maybe this could be moved out into a helper scss file as it is duplicate with src/components/PageList/PageTrash.vue